src/tools/db/importBackup/_pgRestore.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+dropdb frap-dev --user frap
```

Comment:
suggest avoid `-f` for `force` to close open connections (eg. intellij etc might have open connections)